src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketExternalLink.java

```diff
@@ -50,7 +49,7 @@ public String getIconClassName() {
     @CheckForNull
     @Override
     public String getIconFileName() {
-        if (!project.hasPermission(Item.CONFIGURE)) {
+        if (!owner.hasPermission(Item.CONFIGURE)) {
             return null;
         }
```

Comment:
I don't see the point of this CONFIGURE permission check. It was not added in this PR though.

Reply:
This was awhile back but I believe the thinking here was the server the job belongs to was part of the config. That said I agree it's not necessary. Will remove in a separate PR.